npm/package.json

```diff
@@ -14,7 +14,7 @@
     "build-css": "npm-run-all add-stylesheet-dir sass-compile compress-css",
     "build:local": "npm-run-all copy-assets build-css build-ts:prod",
     "build": "npm-run-all copy-assets build-css build-ts:prod",
-    "lint": "eslint src/**/*.ts",
+    "lint": "eslint src/**/*.ts src/**/**/*.ts",
```

Comment:
I think if you use just a single quote then it works recursively.

`"lint": "eslint 'src/**/*.ts'"`

Reply:
Oh, cool. Thanks.